opentimelineio_contrib/adapters/xges.py

```diff
@@ -307,7 +307,8 @@ def to_otio(self):
     def _add_layers(self, timeline, otio_timeline, all_names):
         for layer in timeline.findall("./layer"):
             tracks = self._build_tracks_from_layer_clips(layer, all_names)
-            otio_timeline.tracks.extend(tracks)
+            for track in tracks:
+                otio_timeline.tracks.insert(0, track)
```

Comment:
```python
    otio_timeline.tracks.extend(reversed(tracks))
```

Seems more readable than `insert(0, ...)`

Reply:
That wouldn't add the tracks to the start of the `otio_timeline.tracks` list but it would add them to the end.